haiku-apps/bamkeys/bamkeys-1.0.0.recipe

```diff
@@ -1,44 +1,31 @@
 SUMMARY="Spatially map your display to navigate windows with the keyboard"
-DESCRIPTION="BamKeys let's you break your display up into \
-a grid of zones. Each zone is assigned a keyboard combination. \
-
+DESCRIPTION="BamKeys lets you break your display up into \
+a grid of zones. Each zone is assigned a keyboard combination.
 Whenever you press that keyboard combination, BamKeys will focus \
 the front-most window that intersects with that zone, regardless \
 of the program it belongs to! Pressing the same zone's keyboard \
 combination repeatedly will focus widows further back. If you have \
 three windows that intersect a zone, pressing that zones hotkey three \
-times will cycle through all those windows. \
-
-Configuring - \
+times will cycle through all those windows.
 
+Configuring -
 Open up the BamKeys Preferences, you'll see a little display with a 3x3 \
 grid, a check box, two pop up menus, and a bunch of key cap looking \
-things. \
-
+things.
 The default key mappings for this 3x3 grid are: the left "command" (or alt) \
 key + numeric keypad 1 through 9 (with NumLock ON). If you'd like to \
-change these settings you can - \
-
-• Using the mouse: \
-
-1. Click the zone on the little display you want to change. It will darken. \
-
-2. Press (and release) the key combination you want to use. \
-
-3. Click the zone again to de-select it. \
-
-
-• Using the keyboard: \
-
-1. Tab until the screen is selected. \
-
-2. Press the space-bar. A zone will darken in the grid. \
-
-3. You can use the arrow keys to change the selected zone. \
-
-4. Pressing (and releasing) a keyboard shortcut will \
-set the hotkey for that zone. \
-
+change these settings you can -
```

Comment:
To work with the following "Using the mouse/keyboard", you may want to change this to: "These settings can be changed by:"